crates/oxc_codegen/src/lib.rs

```diff
@@ -105,6 +131,22 @@ impl<const MINIFY: bool> Codegen<MINIFY> {
         unsafe { String::from_utf8_unchecked(self.code) }
     }
 
+    pub fn build_with_sourcemap(
```

Comment:
We don't need a separate method.

I think the API should be a builder pattern `fn with_sourcemap(mut self, yes: bool) {}` with a `sourcemap: bool` in the codegen struct.